addons/website/static/tests/builder/website_builder/highlight.test.js

```diff
@@ -21,6 +21,13 @@ class FakeEditInteractionPlugin extends Plugin {
     stopInteraction() {}
 }
 
+function checkHighlightColor(highlightStyle) {
+    const color = document.documentElement
+        .querySelector(`span.${highlightStyle}`)
+        .style.getPropertyValue("--text-highlight-color");
+    expect(color).toBe("var(--hb-cp-o-color-1)");
+}
+
```

Comment:
I think this seems unnecessary, the color is same do we need to check color again and again ?